### DISCUSSION TITLE
Touch Events Not Working on Mobile and iPad in GrapesJS with Next.js

### DISCUSSION THREAD

**Discussion by prakashfamcom** [GrapesJS/grapesjs#6017] [Q&A]
I'm currently working on a project using GrapesJS with Next.js, and I've encountered an issue where touch events are not functioning properly on mobile devices and iPads. Specifically, I'm unable to drag and drop elements or interact with the editor using touch gestures.

Has anyone experienced similar issues with touch support in GrapesJS, especially in a Next.js environment? If so, how did you resolve it? Any tips, workarounds, or code snippets would be greatly appreciated.

**Comment by artf** [comment]
Try this plugin: https://github.com/GrapesJS/touch

**Comment by prakashfamcom** [comment]
I have tried but not working can you please share code example in nextjs.
 
 
-----Original Message-----
